### ISSUE TITLE
ZIM `Description` should not be required & more

### ISSUE THREAD

**Issue by kelson42** [openzim/nautilus#46]
Look at https://wiki.openzim.org/wiki/Metadata

`LongDescription` should be configurable as well

Would be great to check the max size for both.

**Comment by ankur12-1610** [comment]
Hey @kelson42 I would like to work on this one :)

**Comment by rgaudin** [comment]
@ankur12-1610 please do. Note that issue title was incorrect. It should be required from user.

**Comment by KrishnaRishi2208** [comment]
I am working on this one. Pretty easy though.

**Comment by ankur12-1610** [comment]
> I am working on this one. Pretty easy though.

Ah! sorry @KrishnaRishi2208 I am already working on this one :sweat_smile:

**Comment by KrishnaRishi2208** [comment]
> > I am working on this one. Pretty easy though.
> 
> Ah! sorry @KrishnaRishi2208 I am already working on this one sweat_smile

Wait, so I cant work on this one?? 
I am sorry, I didn't know that once a task is assigned to someone, we are not supposed to work on it.

Will remember next time.

Thanks for informing me.

**Comment by rgaudin** [comment]
Yes, that's what the assignee field is for. If someone's already assigned, skip it or ask if it's been a long time.

**Comment by Onyx2406** [comment]
hey, can I solve this? I have already worked on a similar issue in zim-tools.

**Comment by rgaudin** [comment]
@Onyx2406, let's give a couple days to @ankur12-1610 who already has started a PR. If not, we'll close and reassign as this is really a trivial thing. Please note that it should leverage scraperlib

**Comment by rgaudin** [comment]
@Onyx2406 this is now assigned to you.

**Comment by nithinkatta** [comment]
Hey hi I would like to solve this if @Onyx2406 couldn't.

**Comment by rgaudin** [comment]
No feedback in 5days… @Onyx2406 this task is now assigned to @nithinkatta

**Comment by KrishnaRishi2208** [comment]
@rgaudin If there is no response from @nithinkatta , I would love to solve this issue.

**Comment by KrishnaRishi2208** [comment]
> Hi, I'm actually new to this open-source contribution stuff and enthusiastic about GSOC 2023, can you help me out how to approach this problem so that i will solve the issue by my python coding skills.

You have to first locally download Nautilus python package and make changes to the Nautilus library so that the description will now be mandatory and also you have to add the option for long description.

**Comment by rgaudin** [comment]
I've unassigned the issue. Those interested can can submit PR directly.

Don't work on it If you're worried you may waste your time because someone could submit a correct PR first.

**Comment by KrishnaRishi2208** [comment]
@rgaudin I have made changes to the entrypoint.py file , scraper.py file, init.js file to include the above functionality. I also had to change the filesystem.py file in zimscraperlib/zim to support longdescription while making the zim file. I achieved the functionality and solved the issue, but it wont work unless zimscraperlib/zim/filesystem.py file is changed. Could you please guide me on what to do next.

**Comment by rgaudin** [comment]
I'll release scraperlib tomorrow

**Comment by KrishnaRishi2208** [comment]
> I'll release scraperlib tomorrow

Great, I have created a PR. Please review it.
Thank you
